mkdud

```diff
@@ -282,7 +282,14 @@ if($opt_create) {
     @dists = sort keys %d;
 
     for (@dists) {
-      die "unsupported dist \"$_\"\n" if !/^((leap)?\d+\.\d+|tw|sle[sd]\d+)$/;
+      if(!/^((leap)?\d+\.\d+|tw|sle[sd]\d+)$/) {
```

Comment:
I was more hoping for a hint like "for example, tw, sles-12, sled-12"; "see help" would also be an alternative, but that scroll by for quite a while; so maybe a combination?

"valid examples: sles-12, tw; see --help for more"

?